### PR TITLE
embedded youtube video in description page

### PR DESCRIPTION
**Pull request by schaferyan**
- [x] Embed youtube iframe in description page
- [x] Add `mute`, `loop`, `autoplay` and `modestbranding` parameters to src url
- [x] Edit the id part of src url (just after `embed`) to show the correct video
- [x] Add appropriate css to position and style the video as needed

### CODE REVIEW

File: src/Components/Description.js [hunk 1]
```diff
@@ -57,6 +57,20 @@ export default function Description() {
...
         <Col md='auto'></Col>
       </Row>
+      <Row>
+        <Col md='auto'></Col>
+        <Col className='md-8'>
+          <iframe width="560"
+            height="315"
+            src="https://www.youtube.com/embed/nGCEpUAnkSg?controls=0&autoplay=1&mute=1&modestbranding=1&loop=1"
+            title="YouTube video player"
+            frameBorder="0"
```
Comment (nojronatron) [inline src/Components/Description.js:67]: Looks like this attribute is deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-frameborder

Comment (schaferyan) [reply src/Components/Description.js:67]: Good catch, I'll remove this

**Review by nojronatron — CHANGES_REQUESTED** [review]
Suggesting a few changes:
1. On phone-sized screens, the video runs outside the view port. Try adding minWidth of 330px (likely the smallest phone) and height:auto.
2. IFrame has an objectFit:contain or objectFit:scale-down to force the embeded video to stay a reasonable size.
3. Add a 'rounded-4' to the iframe so it matches the other content elements.
4. The containing Row is missing 'p-2 mb-3' so the IFrame doesn't match the other content elements.
5. Accessibility: Alter the 'title' attribute to be descriptive of the content for example 'YouTube video player demonstrates how to play Lingo Bingo.'

**Comment by schaferyan** [comment]
There seem to be some unexpected complications with resizing iframes. Currently the width resizes with the window, but the height stays the same (at whatever is specified in the HTML height attribute, or the automatic value if you leave it blank). Also, I can't seem to get it to shrink past  a width of about 360px.

**Comment by nojronatron** [comment]
Please take a peek at #62 with some suggestions. Because IFrame's Width and Height attributes require pixel units, there is not an easy way to make the IFrame itself responsive, so I picked a reasonable in-between that will work on phone and isn't too small for desktop. Also, the video has a pop-out button so that is a good excuse to not fight with video size if we don't want to.

**Pull request merged**